server/mentionscrawler/models/mention.py

```diff
@@ -14,6 +14,7 @@ class Mention(db.Model):
     hits = db.Column(db.Integer, nullable=False)
     # using integer for now because the reddit api returns things in unix time
     date = db.Column(db.Integer, nullable=False)
+    sentiment = db.Column(db.Float)
```

Comment:
Why are we allowing this to be nullable? I remember you mentioned something about it, I just don't remember what the reason was.

Reply:
Because the snippet can be null (the sentiment analysis is performed only on the snippet). I'll add nullable=False so it is consistent

Reply:
what search terms are you using that are giving you null snippets, they should never be null... because I'd like to fix that

Reply:
Oh you might have fixed that at some point. I remember in a previous build that there was a mention with no snippet, just the title. I don't know if that snippet was null or maybe it was the empty string.

Reply:
it was probably a previous build, since at one point I had it set to allow just links, which would have given null snippets, but I've since disabled that.